### PR TITLE
 Add license headers to all files

### PR DESCRIPTION
**Pull request by natemcmaster**
Looks like a lot, but it's just code cleanup. Ran R# on whole project.

cc @kichalla

### CODE REVIEW

File: src/MusicStore/Components/ISystemClock.cs [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// See License.txt in the project root for license information
+
+using System;
 
 namespace MusicStore.Components
 {
     /// <summary>
-    /// Abstracts the system clock to facilitate testing.
+    ///     Abstracts the system clock to facilitate testing.
```
Comment (dougbu) [inline src/MusicStore/Components/ISystemClock.cs]: How is this an improvement?

File: src/MusicStore/Controllers/AccountController.cs [hunk 2]
```diff
@@ -151,8 +153,10 @@ public IActionResult Register()
                     // For more information on how to enable account confirmation and password reset please visit http://go.microsoft.com/fwlink/?LinkID=320771
                     // Send an email with this link
                     string code = await UserManager.GenerateEmailConfirmationTokenAsync(user);
-                    var callbackUrl = Url.Action("ConfirmEmail", "Account", new { userId = user.Id, code = code }, protocol: HttpContext.Request.Scheme);
-                    await MessageServices.SendEmailAsync(model.Email, "Confirm your account",
+                    var callbackUrl = Url.Action(
+                        "ConfirmEmail", "Account", new { userId = user.Id, code = code }, protocol: HttpContext.Request.Scheme);
+                    await MessageServices.SendEmailAsync(
+                        model.Email, "Confirm your account",
                         "Please confirm your account by clicking this link: <a href=\"" + callbackUrl + "\">link</a>");
```
Comment (dougbu) [inline src/MusicStore/Controllers/AccountController.cs]: Style is generally "if you wrap, give every argument its own line." [Line 159](https://github.com/aspnet/MusicStore/pull/605/files#diff-d9ee8fb38dcc504d9616e8e6611fae7cR159) looks particularly odd.

File: src/MusicStore/Controllers/AccountController.cs [hunk 3]
```diff
@@ -174,7 +178,8 @@ public IActionResult Register()
         [AllowAnonymous]
         public async Task<ActionResult> ConfirmEmail(string userId, string code)
         {
-            if (userId == null || code == null)
+            if (userId == null
+                || code == null)
```
Comment (dougbu) [inline src/MusicStore/Controllers/AccountController.cs]: Why?

File: src/MusicStore/Startup.cs [hunk 4]
```diff
@@ -36,41 +39,34 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<AppSettings>(Configuration.GetSection("AppSettings"));
 
             var useInMemoryStore = !_platform.IsRunningOnWindows
-                || _platform.IsRunningOnMono
-                || _platform.IsRunningOnNanoServer;
+                                   || _platform.IsRunningOnMono
+                                   || _platform.IsRunningOnNanoServer;
```
Comment (dougbu) [inline src/MusicStore/Startup.cs]: Definitely not. Just wrap to where VS defaults.

**Comment by dougbu** [comment]
- Do we need license headers in these samples?
- Rest looks like churn that I only got about a quarter of the way through. Some changes are negative.

**Comment by natemcmaster** [comment]
These are the same code formatting guidelines we use on other repos via R#.

**Comment by dougbu** [comment]
Most repos don't use R# and the style there is what VS defaults to.

**Pull request closed without merging**